### DISCUSSION TITLE
New install issue on macos 11.6 big sur

### DISCUSSION THREAD

**Discussion by khalrehman** [Homebrew/discussions#2227] [Getting started]
Hi, i am having issues installing on a bran new macos Big sur 11.6. I have tried allsorts but nothing seems to work. Its a strange one in that looks like connections is being dropped during the install. I followed various threads and uninstalled and reinstalled a few times, but no luck.

error trace
=========

 * [new tag]             3.2.8                  -> 3.2.8
 * [new tag]             3.2.9                  -> 3.2.9
HEAD is now at 9b42a104e Merge pull request #12158 from FnControlOption/patch-7
==> Tapping homebrew/core
remote: Enumerating objects: 1048616, done.
remote: Counting objects: 100% (37/37), done.
remote: Compressing objects: 100% (25/25), done.
error: 178 bytes of body are still expected61.36 MiB | 5.69 MiB/s
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: index-pack failed
Failed during: git fetch --force origin refs/heads/master:refs/remotes/origin/master

===========
if check installation via brew -v i get;
Homebrew 3.2.14-35-g9b42a10
Homebrew/homebrew-core (no Git repository)

**Comment by SMillerDev** [comment]
See https://github.com/Homebrew/discussions/discussions/666 for likely solutions. There's unfortunately nothing Homebrew can do for you here.

**Reply by khalrehman** [reply]
I managed to get in installed through numerous retries and then turning off the macos firewall, but I still can't install anything e.g curl or wget

brew install wget
==> Tapping homebrew/cask
Cloning into '/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask'...
remote: Enumerating objects: 594670, done.
remote: Counting objects: 100% (36/36), done.
remote: Compressing objects: 100% (24/24), done.
error: 7201 bytes of body are still expected.14 MiB | 1.77 MiB/s
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: index-pack failed
Error: Failure while executing; `git clone https://github.com/Homebrew/homebrew-cask /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask --origin=origin --template=` exited with 128.


.....brew -v
Homebrew 3.2.14-45-gd6eb9ec
Homebrew/homebrew-core (git revision d6eb9eca5; last commit 2021-10-01)
... brew update
Already up-to-date.
...%

**Reply by carlocab** [reply]
What do `brew config` and `brew doctor` say?

**Reply by khalrehman** [reply]
brew config

HOMEBREW_VERSION: 3.2.14-45-gd6eb9ec
ORIGIN: https://github.com/Homebrew/brew
HEAD: d6eb9eca52058dfccb53b74fa2ea6f2bf0aecb52
Last commit: 81 minutes ago
Core tap ORIGIN: https://github.com/Homebrew/brew
Core tap HEAD: d6eb9eca52058dfccb53b74fa2ea6f2bf0aecb52
Core tap last commit: 81 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 16
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 16-core 64-bit kabylake
Clang: 13.0.0 build 1300
Git: 2.30.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.6-x86_64
CLT: 13.0.0.0.1.1630607135
Xcode: N/A

**Reply by khalrehman** [reply]
brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Homebrew/homebrew-core was not tapped properly! Run:
  rm -rf "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core"
  brew tap homebrew/core

**Reply by carlocab** [reply]
Have you tried
```
rm -rf "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core"
brew tap homebrew/core
```
?

**Reply by khalrehman** [reply]
I ran the recommendations and same issue....

..... % rm -rf "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core"
..... % brew tap homebrew/core
==> Tapping homebrew/core
Cloning into '/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core'...
remote: Enumerating objects: 1048692, done.
remote: Counting objects: 100% (111/111), done.
remote: Compressing objects: 100% (78/78), done.
error: 6216 bytes of body are still expected5.88 MiB | 5.08 MiB/s
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: index-pack failed
Error: Failure while executing; `git clone https://github.com/Homebrew/homebrew-core /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core --origin=origin --template=` exited with 128.

**Reply by khalrehman** [reply]
Guys anything else I can try to do or is this a specific issue on git/macos 11.6?

**Reply by carlocab** [reply]
It's most likely a specific issue with your network connection. I installed Homebrew on macOS 11.6 with no issues just today. You can read through #666 for things other people have tried.

**Reply by khalrehman** [reply]
Ok thanks @carlocab. I'll try a few  other things.

**Reply by khalrehman** [reply]
Issue was with my Virgin Media router. Once  I used my mobile hotspot on my phone everything has worked seamlessly!! On to my router config and my broadband provider.
